test/helpers.c

```diff
@@ -67,4 +67,14 @@ struct notification *test_notification_with_icon(const char *name, gint64 timeou
         return n;
 }
 
+void pango_memory_cleanup()
```

Comment:
Is this still needed with the suppression? I prefer not adding a fontconfig dependency for a test.